modules/dream2nix/pip/default.nix

```diff
@@ -117,12 +117,11 @@ in {
     l.mapAttrs (_: l.mkOverride 1002) {
       # This is imported directly instead of depending on dream2nix.packages
       # with the intention to keep modules independent.
-      fetchPipMetadataScript = import ../../../pkgs/fetchPipMetadata/script.nix {
-        inherit lib;
+      fetchPipMetadataFunction = nixpkgs.callPackage ../../../pkgs/fetchPipMetadata/script.nix;
+      fetchPipMetadataArgs = {
```

Comment:
This split between function and args is because I've not found a way to get the "super" definition of a dep.

It seem due to the modular nature of dream2nix, but something like this yields an infinite recursion:

```nix
{config, ...}: {
  deps = {nixpkgs, ...}:
    fetchPipMetadataScript = config.deps.fetchPipMetadataScript.override {env.PIP_FIND_LINKS = "${config.deps.setuptools-odoo.out.dist}";};
  };
}
```

If there were a way to do that, just wrapping the script with `callPackage` would be enough.